doc/source/operations/cloudkitty.rst

```diff
@@ -0,0 +1,125 @@
+==========
+CloudKitty
+==========
+
+Configuring in kayobe-config
+============================
+
+By default, CloudKitty uses Gnocchi and Ceilometer as the collector and fetcher
+backends. Unless the system has a specific reason not to, we recommend instead
+using Prometheus as the backend for both. The following instructions explain
+how to do this.
+
+Enable CloudKitty and disable InfluxDB, as we are using OpenSearch as the
+storage backend. Set the following in ``kolla.yml``:
+
+.. code-block:: console
+
+  kolla_enable_cloudkitty: true
+  # Explicitly disable influxdb as we are using OpenSearch as the CloudKitty backend
+  kolla_enable_influxdb: false
+
+Set Prometheus as the backend for both the collector and fetcher, and Elasticsearch as the
+storage backend. Note that this is patched in our fork of CloudKitty to also
+work with OpenSearch. Proper support is still pending in Kolla-Ansible `here
+<https://review.opendev.org/c/openstack/kolla-ansible/+/898555>`__. If you have
+TLS enabled, you will also need to allow insecure connections for Prometheus
+and Elasticsearch. Set the following in ``kolla/globals.yml``:
+
+.. code-block:: console
+
+  cloudkitty_collector_backend: prometheus
+  cloudkitty_fetcher_backend: prometheus
+  cloudkitty_storage_backend: elasticsearch
+  cloudkitty_elasticsearch_insecure_connections: true
+  cloudkitty_prometheus_insecure_connections: true
```

Comment:
```suggestion
Set Prometheus as the backend for both the collector and fetcher, and Elasticsearch as the
storage backend; this is achieved by configuring the following within ``kolla/globals.yml``:

.. code-block:: console

  cloudkitty_collector_backend: prometheus
  cloudkitty_fetcher_backend: prometheus
  cloudkitty_storage_backend: elasticsearch

Note that in our fork of CloudKitty `cloudkitty_storage_backend:` is patched to additionally accept `opensearch` as the storage backend. Proper support of this is still pending for Kolla-Ansible `here
<https://review.opendev.org/c/openstack/kolla-ansible/+/898555>`__. Furthermore, if you have TLS enabled, you will also need to allow insecure connections for Prometheus
and Elasticsearch. 

.. code-block:: console

  cloudkitty_elasticsearch_insecure_connections: true
  cloudkitty_prometheus_insecure_connections: true
```

Reply:
Seems sensible to split these. Will make the change.